img2dataset/resizer.py

```diff
@@ -24,6 +24,7 @@ class ResizeMode(Enum):
     keep_ratio = 1  # pylint: disable=invalid-name
     center_crop = 2  # pylint: disable=invalid-name
     border = 3  # pylint: disable=invalid-name
+    bounding_box = 4 # pylint: disable=invalid-name
```

Comment:
thanks, let me think.

Reply:
@rom1504 If we have to support both, we will need to introduce a separate flag I suppose instead of providing this new feature via resize_mode.
But, we may need to provide an option to specify the order of both of these operations then. 
I can imagine below scenario:
* someone downloads the initial dataset with one of the existing resize modes
* they do bounding box computations on top of resized image and prepares the metadata files for distributing

Reply:
I think it's clear the resizing should happen after the cutting in boxes